content/en/tracing/troubleshooting/tracer_debug_logs.md

```diff
@@ -88,6 +88,7 @@ const tracer = require('dd-trace').init({
   debug: true
 })
 ```
+**Note:** For version 2.X debug mode can only be configured by the environment variable `DD_TRACE_DEBUG=true`.
```

Comment:
Can we do the opposite and make the doc all for v2 and then the note would simply note the deprecation of the option that was available in v1?

Reply:
@rochdev Is this more what you had in mind?

To enable debug mode for the Datadog Node.js Tracer, use the environment variable `DD_TRACE_DEBUG=true`. 

**Note:** For versions below 2.X, debug mode can be enabled programmatically inside the tracer initialization.
```javascript

const tracer = require('dd-trace').init({
  debug: true
})
```

Reply:
Probably more along the line of "For versions below 2.X, debug mode could be enabled programmatically inside the tracer initialization but this is no longer available.". And I would remove the code example. The reasoning is that when code exists in any documentation, some users end up copying that code even it's explicitly stated to not use it, so it's better to avoid code completely and just have a note for users who are upgrading.

Reply:
Just updated the language to reflect this